| -> Erreure: : fin de boucle
| File "in_python_code.py", line 11
| for loop in range(5)
                       ^
| SyntaxError: invalid syntax

| -> Erreure: : fin de la condition
| File "in_python_code.py", line 8
|     if 0 == 0
            ^
| SyntaxError: invalid syntax

| -> Erreure des : pour tous les blocs: if, elif, else, for, while, class, or def

| -> Erreure: = insted of ==
| File "in_python_code.py", line 9
|     if a = 0:
|          ^
| SyntaxError: invalid syntax

| -> Erreure: Parenthès ouvrantes/fermantes
| File "in_python_code.py", line 10
|     d = (5+)
|     ^
| SyntaxError: invalid syntax

-> Erreure: Autres syntaxes invalides
File "in_python_code.py", line 10
    d = (5+)
           ^
SyntaxError: invalid syntax
--------- > Parenthès non fermées: vérifier si elle ont bien étée fermée auparavant !

| -> => insted of >=
| File "in_python_code.py", line 13
|     if f => 0:
|          ^
| SyntaxError: invalid syntax

| -> =< instead of <=
| File "in_python_code.py", line 13
|     if f =< 0:
|          ^
| SyntaxError: invalid syntax

| -> Ouverture/fermeture guillemets  (avec '' et "")
| File "in_python_code.py", line 30
|     myText = "Salut !
|                     ^
| SyntaxError: EOL while scanning string literal

=====================================================================================================================================

| -> Indentation error: pas la même indentation
| File "in_python_code.py", line 18
|     for i in range(2):
|                      ^
| IndentationError: unindent does not match any outer indentation level

| -> Erreur: pas d'indentation/contenu dans un bloc
| File "in_python_code.py", line 16
|     print(loop)
|         ^
| IndentationError: expected an indented block

| -> Indice inexistant dans un tableau
| Traceback (most recent call last):
|   File "in_python_code.py", line 27, in <module>
|     print(a[9])
| IndexError: list index out of range
|
| -> Clée inexistante dictionaire:
| Traceback (most recent call last):
|   File "in_python_code.py", line 32, in <module>
|     print(b['l'])
| KeyError: 'l'

| -> Division par 0
|   File "in_python_code.py", line 30, in diviser
|     print(1/a)
| ZeroDivisionError: integer division or modulo by zero

| -> Import inexistant
| Traceback (most recent call last):
|   File "in_python_code.py", line 1, in <module>
|     from robot import *
| ImportError: No module named robot
